### PR TITLE
week10 progress - Outreachy internship experience

### PR DESCRIPTION
**Pull request by deborahtrez**
This is my week 10 progress report

### CODE REVIEW

File: content/news/2021-02-07-week-10-progess.md [hunk 1]
```diff
@@ -19,9 +19,9 @@ This week:
...
-`Start the Mixxx software`
-`Import music to the Mixxx library - import your audio files`
-`Analyze your library tracks (what it means to analyze a track)`
-`Configure Sound output - especially if you're not using any DJ Hardware`
-`Play your first track with Mixxx`
+`Start the Mixxx software` <br>
+`Import music to the Mixxx library - import your audio files` <br>
+`Analyze your library tracks (what it means to analyze a track)` <br>
+`Configure Sound output - especially if you're not using any DJ Hardware` <br>
+`Play your first track with Mixxx` <br>
```
Comment (Swiftb0y) [inline content/news/2021-02-07-week-10-progess.md]:
```suggestion
  - Start the Mixxx software
  - Import music to the Mixxx library - import your audio files 
  - Analyze your library tracks (what it means to analyze a track) 
  - Configure Sound output - especially if you're not using any DJ Hardware 
  - Play your first track with Mixxx 
```
You can make nested lists in markdown by prefixing every line with two extra spaces.
https://commonmark.org/help/tutorial/10-nestedLists.html

Comment (Holzhaus) [reply content/news/2021-02-07-week-10-progess.md]: Also, the `<br>` tags are unneeded and backticks are for code.

Comment (deborahtrez) [reply content/news/2021-02-07-week-10-progess.md]: Thank you! :)

File: content/news/2021-02-07-week-10-progess.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+title: "About my Outreachy internship with Mixxx DJ: Week 10 progress"
+authors: Aanyu Deborah Oduman
+tags: personal, outreachy, internship, manual
+date: 2021-02-07 23:09:00
+
+_This is the tenth weekly progress report of my Outreachy internship with Mixxx DJ._
+
+This week:
+
+- I made the required edits to the YouTube tutorial about the Mixxx  communities and uploaded it to YouTube.
```
Comment (Holzhaus) [inline content/news/2021-02-07-week-10-progess.md]:
```suggestion
- I made the necessary edits to the YouTube tutorial about the Mixxx community and uploaded it to YouTube.
```

File: content/news/2021-02-07-week-10-progess.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+ Here is the link. [Get in touch with the Mixxx community](https://www.youtube.com/watch?v=0f6QrjiFNJQ)
```
Comment (Holzhaus) [inline content/news/2021-02-07-week-10-progess.md]:
```suggestion
 [Here is the link.](https://www.youtube.com/watch?v=0f6QrjiFNJQ)
```

File: content/news/2021-02-07-week-10-progess.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+- I had a brief video call with community team leaders from the [Tor Project](https://www.torproject.org/) to discuss more about the [internet censorship situation](https://explorer.ooni.org/country/UG) that has been going on in Uganda.
+ The purpose of the discussion was to understand more about how I am using Tor and other anti-censorship tools, and how it can be improved based on my feedback.
+- As part of the homework that I did for the cookbook chapter, I read through the first few chapters of the Mixxx manual, looking for sections that could use the step by step instruction approach that the cookbook will provide.
```
Comment (Holzhaus) [inline content/news/2021-02-07-week-10-progess.md]: "Homework" sounds like school exercises.

Perhaps rephrase it:
```suggestion
- To get some inspiration for the cookbook chapter, I read through the first few chapters of the Mixxx manual, looking for sections that could use the step by step instruction approach that the cookbook will provide.
```

**Review by Holzhaus — COMMENTED** [review]
Thanks!

**Review by ywwg — APPROVED** [review]
lgtm!

**Comment by Holzhaus** [comment]
@ywwg You forgot to press merge :D

**Pull request merged**